docs_src/pipelinerun.md

```diff
@@ -178,9 +122,68 @@ The output in results/pipeline0.log file lists FPS (frames per second) during pi
 ...
 ```
 
-The results/ directory is volume mounted to the pipeline container, the log files inside the results/ directory will keep on growing as the pipeline is still running, you can call ./stop_all_docker_containers.sh to stop the pipeline and all running containers.
+The **results** directory is volume mounted to the pipeline container. The log files within the **results** increase as the pipeline continues to run. You can [stop the pipeline](/pipelinerun.md#stop-pipeline-run) and the containers that are running.
+
+**Failure**
+Review the console output for errors if you do not see all the Docker* containers. Sometimes dependencies fail to resolve. Address obvious issues and [rerun the pipeline](/pipelinerun.md#Rstart-pipeline).
+    
+---
+    
+### Stop Pipeline
+
+Run the following command to stop the pipeline and the containers that are running:
+    
+```
+./stop_all_docker_containers.sh
+```
+
+---
+## Run Camera Simulator
```

Comment:
What is the need for running a camera simulator along with the pipeline? It would be good to add an explanation

Reply:
In case users don't have camera device plugged into the system, they can run the camera simulator to view the pipeline analytic results based on a sample video file to mimic real time camera video.

Reply:
Additionally the camera simulator can be used to infinitely loop through a video file for consistent benchmarking. For example if you want to validate that the performance is the same for 6 hours, 12 hours, and 24 hours. Looping the same video should produce the same results no matter how long you run.